bowtie/schemas/benchmark_report.json

```diff
@@ -120,16 +121,30 @@
       "type": "object",
       "description": "Contains the Benchmark results for the Benchmark Group.",
       "title": "Benchmark Group Result",
-      "required": ["name", "description", "benchmark_results"],
+      "additionalProperties": false,
+      "required": [
+        "name",
+        "benchmark_type",
+        "description",
+        "varying_parameter",
+        "benchmark_results"
+      ],
       "properties": {
         "name": {
           "description": "Benchmark Group's Name.",
           "type": "string"
         },
+        "benchmark_type": {
+          "description": "The type of the Benchmark Group (Eg. keyword, default, etc.)",
```

Comment:
Should this be an enum? What does `etc.` mean here? Do we use arbitrary values, or is this basically a preliminary implementation of #1388 ?

Reply:
Yeah, a preliminary implementation of #1388, this should improve going on further. etc. because currently we just have 2 types but going further we could have others too.

Reply:
Got it, sounds good!